message KnownHosts{
	repeated uint32 name=1;
}

message Subscription
{
	required uint32 host=1;
	required uint32 topicid=2;
}
message HostSubscriptions{
	repeated Subscription topics=1;
}


